gef-ui/server/controllers/unissued-facilities/validation.test.js

```diff
@@ -345,6 +345,296 @@ describe('validation()', () => {
     expect(result.aboutFacilityErrors).toEqual(expectedFacilityErrors);
   });
 
+  it('should return object with errors populated if issue date has errors', async () => {
```

Comment:
Can abstract the test cases to more illustrative blocks i.e. 
* should return error when date is blank...
* Should return error when date has symbol..

Can we please add a test case for either date/month/year to be blank please.

Reply:
done